cli/client.go

```diff
@@ -837,12 +838,34 @@ var clientRetrieveCmd = &cli.Command{
 			Path:  cctx.Args().Get(1),
 			IsCAR: cctx.Bool("car"),
 		}
-		if err := fapi.ClientRetrieve(ctx, offer.Order(payer), ref); err != nil {
-			return xerrors.Errorf("Retrieval Failed: %w", err)
+		updates, err := fapi.ClientRetrieve(ctx, offer.Order(payer), ref)
+		if err != nil {
+			return xerrors.Errorf("error setting up retrieval: %w", err)
 		}
 
-		fmt.Println("Success")
-		return nil
+		for {
+			select {
+			case evt, chOpen := <-updates:
+
+				fmt.Printf("Retrieval Event: %s, State: %s, BytesReceived: %d, PaymentSent: %s\n",
```

Comment:
I would say hide this behind a verbose flag, but then again, it might be useful (at least for now) to always have this print out in order for us to help users debug things.